diff -dur bacula-web-6.0.0.orig/core/global.inc.php bacula-web-6.0.0/core/global.inc.php
--- bacula-web-6.0.0.orig/core/global.inc.php	2013-11-11 17:06:40.000000000 +0100
+++ bacula-web-6.0.0/core/global.inc.php	2013-12-06 09:45:58.000000000 +0100
@@ -31,20 +31,21 @@
  // Views path
  define('BW_ROOT', getcwd());
  define('VIEW_DIR', BW_ROOT . "/application/view/");
- define('VIEW_CACHE_DIR', "application/view/cache");
+ define('VIEW_CACHE_DIR', '/var/cache/bacula-web');
+ define('VIEW_CACHE_URL', 'cache');
  
  // Configuration
- define('CONFIG_DIR', BW_ROOT . "/application/config/");
+ define('CONFIG_DIR', '/etc/webapps/bacula-web/');
  define('CONFIG_FILE', CONFIG_DIR . "config.php");
  
  // Locales
  define('LOCALE_DIR', BW_ROOT . '/application/locale');
  
  // Smarty
- require_once(BW_ROOT . '/core/external/smarty-gettext/smarty-gettext.php');
+ require_once('/usr/share/php/Smarty/Smarty.class.php');
  
  // PHPLot
- require_once(BW_ROOT . '/core/external/phplot/phplot.php');
+ require_once('/usr/share/php/phplot/phplot.php');
  
  // Constants
  require_once(BW_ROOT . '/core/const.inc.php');
diff -dur bacula-web-6.0.0.orig/core/graph/cgraph.class.php bacula-web-6.0.0/core/graph/cgraph.class.php
--- bacula-web-6.0.0.orig/core/graph/cgraph.class.php	2013-11-11 17:06:40.000000000 +0100
+++ bacula-web-6.0.0/core/graph/cgraph.class.php	2013-12-06 09:40:53.000000000 +0100
@@ -32,6 +32,7 @@
 
     function __construct($filename = "graph.png") {
         $this->img_filename = VIEW_CACHE_DIR . '/' . $filename;
+        $this->img_fileurl = VIEW_CACHE_URL . '/' . $filename;
     }
 
     public function SetData($data_in, $graph_type) {
@@ -52,7 +53,7 @@
     }
 
     private function get_Filepath() {
-		return $this->img_filename;
+		return $this->img_fileurl;
     }
 	
     // ==================================================================================
Only in bacula-web-6.0.0/core/graph: cgraph.class.php.orig
diff -dur bacula-web-6.0.0.orig/core/i18n/ctranslation.class.php bacula-web-6.0.0/core/i18n/ctranslation.class.php
--- bacula-web-6.0.0.orig/core/i18n/ctranslation.class.php	2013-11-11 17:06:40.000000000 +0100
+++ bacula-web-6.0.0/core/i18n/ctranslation.class.php	2013-12-06 09:40:53.000000000 +0100
@@ -32,9 +32,6 @@
     }
 
     function set_Language(&$template) {
-        // Template engine block registration
-        $template->register_block('t', 'smarty_translate');
-
         // Setting up language
         putenv("LANGUAGE=" . $this->language . '.' . $this->charset);
         putenv("LANG=" . $this->language . '.' . $this->charset);
